app/models/token.py

```diff
@@ -0,0 +1,21 @@
+import uuid
+from datetime import datetime
+
+from sqlmodel import SQLModel
+
+
+class TokenModel(SQLModel):
+    token: str
+
+    @classmethod
+    def from_str(cls, key: str) -> "TokenModel":
+        return cls(token=key)
+
+
+class TokenPayload(SQLModel):
+    sub: str
+    exp: datetime
+    iat: datetime
```

Comment:
Correcciones:
- Siguiendo las convenciones, se deberia llamar TokenPublic
- Comentar qué significa cada variable en el código y el estandar que siguen.
```suggestion
class TokenPublic(SQLModel):
    sub: str # Usuario: user_public_id
    exp: datetime # Expiration timestamp
    iat: datetime # Creation timestamp
```

Reply:
Es verdad que se pude ver la carga del token pero me parece mal cambiarlo a "Public", Tambien Tomas esta en contra de los comentarios.
Aun asi realizare los cambios